packages/legend-query-builder/src/stores/QueryBuilderResultState.ts

```diff
@@ -129,27 +131,42 @@ export class QueryBuilderResultState {
         !this.queryBuilderState.isParameterSupportDisabled &&
         this.queryBuilderState.parametersState.parameterStates.length
       ) {
-        const letlambdaFunction = buildParametersLetLambdaFunc(
-          this.queryBuilderState.graphManagerState.graph,
-          this.queryBuilderState.parametersState.parameterStates,
-        );
-        const letRawLambda = buildRawLambdaFromLambdaFunction(
-          letlambdaFunction,
-          this.queryBuilderState.graphManagerState,
-        );
-        // reset paramaters
-        if (Array.isArray(query.body) && Array.isArray(letRawLambda.body)) {
-          letRawLambda.body = [
-            ...(letRawLambda.body as object[]),
-            ...(query.body as object[]),
-          ];
-          query = letRawLambda;
-        }
+        query.parameters =
```

Comment:
why is this logic needed.
We should now correctly build parameters in the 
`buildLambdaFunction`, I would expect us only to edit the executionInput with the parameter values